### PR TITLE
🐍🐍🐍 - Adding python files for os commands without piping

### PR DESCRIPTION
**Pull request by liqowomo**
# Adding Python Command with the following code 

# Python code 

```py 
#!/usr/bin/env python3
'''
The following fish shell code 

#!/bin/fish 
function gpx 
    for i in (seq 1 $argv) 
        echo "----------------------------"
        echo "Commit Number" $i
        git add .
        git commit --allow-empty --allow-empty-message -m "$i"
        git push --progress
        echo "----------------------------"
    end 
end

Convert bastard to python

'''

import subprocess

# git add . command - this should be normal one
subprocess.run(["git", "add", "."])

# git commit --allow-empty --allow-empty-message -m ""
subprocess.run(["git", "commit", "--allow-empty",
               "--allow-empty-message", "-m", "🐍🐍🐍 Push with python 🐍🐍🐍"])

# git push --progress && git log --oneline --decorate --graph -10
subprocess.run(["git", "push", "--progress"])
subprocess.run(["git", "log", "--oneline", "--decorae", "--graph", "-10"])

```

# Problems to explore 

- If I store as a variable `subprocess.<commad>`  then output doesnt really get stored in the variable 
- Need to look at some other method of doing that 

# Sniff 

<p align="center">

</p>

### CODE REVIEW

File: frgp.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+# git add . command - this should be normal one
+subprocess.run(["git", "add", "."])
+
+# git commit --allow-empty --allow-empty-message -m ""
+subprocess.run(["git", "commit", "--allow-empty",
+               "--allow-empty-message", "-m", "🐍🐍🐍 Push with python 🐍🐍🐍"])
+
+# git push --progress && git log --oneline --decorate --graph -10
+subprocess.run(["git", "push", "--progress"])
+subprocess.run(["git", "log", "--oneline", "--decorae", "--graph", "-10"])
```
Comment (sniffowomo) [inline frgp.py:32]: This is tight , later need to see how to make sure it can be piped

File: frgp.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python3
+'''
+The following fish shell code
```
Comment (sniffowomo) [inline frgp.py:3]: This shit is long

**Review by sniffowomo — COMMENTED** [review]
Check it son

File: frgp.py (hunk 1 above)
Comment (liqowomo) [reply frgp.py:32]: Ok will addingtong

File: frgp.py (hunk 2 above)
Comment (liqowomo) [reply frgp.py:3]: Aint it fun sin

**Comment by sniffowomo** [comment]
OK LGTM nice additions too 

```ml
No one really knows that thing
```
<p align="right">

</p>

**Comment by sniffowomo** [comment]
Very nice

**Pull request merged**